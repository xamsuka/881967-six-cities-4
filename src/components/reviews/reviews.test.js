import React from 'react';
import renderer from 'react-test-renderer';
import Reviews from './reviews.jsx';

const commentsOffer = [
  {
    comment: `Бла бла бла`,
    date: new Date(`2020-08-04T10:53:33.549Z`),
    id: 1,
    rating: 5,
    user: {
      avatarUrl: `link`,
      id: 1,
      isPro: false,
      name: `Vlad`,
    },
  },
  {
    comment: `Бла бла бла`,
    date: new Date(`2020-08-04T10:53:33.549Z`),
    id: 1,
    rating: 5,
    user: {
      avatarUrl: `link`,
      id: 1,
      isPro: false,
      name: `Vlad`,
    },
  }
];


test(`<Reviews /> render`, () => {
  const three = renderer
    .create(<Reviews reviews = {commentsOffer} />).toJSON();

  expect(three).toMatchSnapshot();
});
