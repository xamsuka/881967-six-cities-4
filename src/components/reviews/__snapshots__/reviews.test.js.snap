// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reviews /> render 1`] = `
<section
  className="property__reviews reviews"
>
  <h2
    className="reviews__title"
  >
    Reviews · 
    <span
      className="reviews__amount"
    >
      2
    </span>
  </h2>
  <ul
    className="reviews__list"
  >
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height={54}
            src="link"
            width={54}
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Vlad
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "100%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          Бла бла бла
        </p>
        <time
          className="reviews__time"
          dateTime={2020-08-04T10:53:33.549Z}
        >
          August 2020
        </time>
      </div>
    </li>
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height={54}
            src="link"
            width={54}
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Vlad
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "100%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          Бла бла бла
        </p>
        <time
          className="reviews__time"
          dateTime={2020-08-04T10:53:33.549Z}
        >
          August 2020
        </time>
      </div>
    </li>
  </ul>
  
</section>
`;
